spec/spec_helper.rb

```diff
@@ -1,3 +1,8 @@
 # frozen_string_literal: true
 
+# rubocop:disable Style/RescueModifier
+
 require 'helpers/match_helper'
+require 'byebug' rescue nil
```

Comment:
Now `byebug` will be included if installed.  On CI runs it is not installed, so this will explode but the rescue will silence it.